Stratis.Bitcoin/Features/Consensus/PosConsensusValidator.cs

```diff
@@ -322,10 +330,14 @@ private static long FutureDriftV1(long nTime)
             return nTime + 10 * 60;
         }
 
-        // TODO: https://github.com/stratisproject/StratisBitcoinFullNode/issues/383
+        private static bool IsDriftReduced(long nTime)
```

Comment:
This was introduced as a fix to address the specific issue of the stratis chain (future drift bug)
Bear in mind as for other chain it wont be necessarily to make the check.

Reply:
Sorry, I don't understand what you mean. Are you talking about BitcoinD node? If so, this code is only called from PosConsensusValidator, which is not part of Bitcoin chain validation. Or are you talking about something else?

Reply:
No I mean this condition is only correct for the stratis chain, on stratis testnet for example it is not needed, as we dont have blocks with the wrong future drift.

Reply:
Yes, I've checked it against testnet, it works correctly as is because our testnet is younger than the condition timestamp.